Marlin/ultralcd.cpp

```diff
@@ -2518,12 +2520,32 @@ void kill_screen(const char* lcd_msg) {
     void _goto_tower_z() { _man_probe_pt(cos(RADIANS( 90)) * delta_calibration_radius, sin(RADIANS( 90)) * delta_calibration_radius); }
     void _goto_center()  { _man_probe_pt(0,0); }
 
+    void _delta_G33_settings() {
```

Comment:
Using the edit functionally without being able to change the value (..., &var, var, var) to show the current settings on LCD